modules/ROOT/pages/configure-cidr-range.adoc

```diff
@@ -0,0 +1,28 @@
+= Configure Pod CIDR Range for Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To configure the pod Classless Inter-Domain Routing (CIDR) range on an existing Anypoint Runtime Fabric cluster:
+
+. Create a resource definition using a new `podCIDR` value.
++
+In the following example, the yaml file is named `pod-cidr.yaml` and `x.x.x.x/16` represents the new pod CIDR range:
++
+```
+kind: clusterconfiguration
```

Comment:
`ClusterConfiguration`? Is it case sensitive? Better stick to the convention in gravity docs: https://gravitational.com/gravity/docs/config/#general-cluster-configuration